nano/lib/config.hpp

```diff
@@ -49,6 +50,15 @@ uint8_t get_minor_node_version ();
 uint8_t get_patch_node_version ();
 uint8_t get_pre_release_node_version ();
 
+std::string GetEnvOrDefault (const std::string & variable_name, const std::string & default_value);
+uint64_t GetEnvThresholdOrDefault (const std::string & variable_name, const uint64_t & default_value);
```

Comment:
Should we be using camel case or lowercase with underscores instead on variables/functions?

Reply:
switched back to underscore